docs/modules/scoring/mdscoring.rst

```diff
@@ -5,3 +5,8 @@ MD Scoring module
    :members:
    :show-inheritance:
    :inherited-members:
+
+MD Scoring default parameters
```

Comment:
all the other modules do not show their name before "default parameters". It would be better to be consistent, right?